.gitignore

```diff
@@ -6,4 +6,5 @@ cypress/downloads
 cypress/screenshots
 cypress/videos
 dist
+mx-sso-api-proxy-rc.yaml
```

Comment:
Is this a config file or something? Is is something we should document?

Reply:
I answered my own question, seems like that is from the sso api proxy. Should we add a link to that `https://gitlab.mx.com/mx/widget-sso-api-proxy#configuration` in the example readme?

Reply:
It's the config for https://github.com/mxenabled/sso-api-proxy. That tool will walk the user through how to configure it, but maybe we link to that project in the example app's readme.

Reply:
Yup, thinking the same.

Reply:
Done @sambev